### PR TITLE
Raw app package name

### PR DESCRIPTION
**Pull request by sparshadotel**
Modified changes as said by @CuriousSugam

### CODE REVIEW

File: app/src/main/java/com/lftechnology/activitylogger/MainActivity.java [hunk 1]
```diff
@@ -2,12 +2,24 @@
...
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        checkPermissionForAccess()
+    }
+
+    /**
+     * if you havent enabled permission, settings opens for granting permission access with this
+     */
+    public void checkPermissionForAccess(){
```
Comment (manas-raj-shrestha) [inline app/src/main/java/com/lftechnology/activitylogger/MainActivity.java]: @darmagedon why is this public?

File: app/src/main/java/com/lftechnology/activitylogger/MyUsageStats.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import android.content.SharedPreferences;
+import android.util.Log;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.List;
+
+
+/**
+ * Created by sparsha on 6/29/2016.
```
Comment (manas-raj-shrestha) [inline app/src/main/java/com/lftechnology/activitylogger/MyUsageStats.java]: @darmagedon write a class description

File: app/src/main/java/com/lftechnology/activitylogger/MyUsageStats.java [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+ */
+public class MyUsageStats {
+    private static final SimpleDateFormat date = new SimpleDateFormat();
+    private static int interval;
+
+    public enum Interval{
```
Comment (manas-raj-shrestha) [inline app/src/main/java/com/lftechnology/activitylogger/MyUsageStats.java]: @darmagedon better create a class called constants and include this enum there

File: app/src/main/java/com/lftechnology/activitylogger/MyUsageStats.java [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.List;
+
+
+/**
+ * Created by sparsha on 6/29/2016.
+ */
+public class MyUsageStats {
```
Comment (manas-raj-shrestha) [inline app/src/main/java/com/lftechnology/activitylogger/MyUsageStats.java]: @darmagedon make class name more relevant

File: app/src/main/java/com/lftechnology/activitylogger/MyUsageStats.java [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+    private static final SimpleDateFormat date = new SimpleDateFormat();
```
Comment (manas-raj-shrestha) [inline app/src/main/java/com/lftechnology/activitylogger/MyUsageStats.java]: @darmagedon i dont think date is a good instance name for SimpleDateFormat.

File: app/src/main/java/com/lftechnology/activitylogger/MyUsageStats.java [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+    public enum Interval{
+        DAILY(0),WEEKLY(1),MONTHLY(2),YEARLY(3),BEST(4);
+        public int value;
+        Interval(int mValue){
+            value = mValue;
+        }
+    }
+
+    public static List<UsageStats> getUsageStatsAppList(Context context){
```
Comment (manas-raj-shrestha) [inline app/src/main/java/com/lftechnology/activitylogger/MyUsageStats.java]: @darmagedon please write the method description

File: app/src/main/java/com/lftechnology/activitylogger/MyUsageStats.java [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+        Log.d("LOG","Date Start:\t"+ date.format(startTime));//TODO remove
+        Log.d("LOG","Date End:\t"+ date.format(endTime));//TODO remove
+        List<UsageStats> usageStatsList =
+                usageStatsManager.queryUsageStats(interval,startTime,endTime);
+        return usageStatsList;
+    }
+
+
+    /**
+     * This Method Should be called for the proper implementation of the class
```
Comment (manas-raj-shrestha) [inline app/src/main/java/com/lftechnology/activitylogger/MyUsageStats.java]: @darmagedon this is not a proper description. mention what this method does.

Comment (sparshadotel) [inline app/src/main/java/com/lftechnology/activitylogger/MyUsageStats.java]: @manas-raj-shrestha  done

**Pull request closed without merging**